hal/include/hal/PinNameAliases.h

```diff
@@ -86,12 +86,35 @@
 #define ARDUINO_UNO_I2C_SDA ARDUINO_UNO_D14
 #define ARDUINO_UNO_I2C_SCL ARDUINO_UNO_D15
 
+// Legacy I2C aliases
```

Comment:
This has to be protected by 
#ifdef TARGET_FF_ARDUINO_UNO

as targets which are not compliant with pin standard are already defining I2C_SDA in enum